src/assets/Generator.Shared/PageableHelpers.cs

```diff
@@ -19,22 +20,24 @@ internal static class PageableHelpers
         private static readonly byte[] DefaultItemPropertyName = Encoding.UTF8.GetBytes("value");
         private static readonly byte[] DefaultNextLinkPropertyName = Encoding.UTF8.GetBytes("nextLink");
 
+        public delegate T ValueFactory<T>(JsonElement element, ModelSerializerOptions? options = default);
```

Comment:
well this is good for readability, if we could do the same for other delegates in the Create(Async)Pageable helper methods would also be great.
But I assume we could do that later since everything is good to work

Reply:
Do you mind opening a tracking item for this in azure-sdk-for-net since this file has moved with all other shared source?

https://github.com/Azure/azure-sdk-for-net/pull/38399